book/src/user/target-tier-policies.md

```diff
@@ -0,0 +1,288 @@
+# Target Tier Policy
+
+## Table of Contents
+
+- [Target Tier Policy](#target-tier-policy)
+  - [Table of Contents](#table-of-contents)
+  - [General](#general)
+  - [Tier 3 target policy](#tier-3-target-policy)
+  - [Tier 2 target policy](#tier-2-target-policy)
+  - [Tier 1 target policy](#tier-1-target-policy)
+
+## General
+
+Zebra provides three tiers of target support:
+
+- Zebra provides no guarantees about tier 3 targets; they exist in the codebase,
+  but may or may not build.
+- Zebra's continuous integration checks that tier 2 targets will always build,
+  but they may or may not pass tests.
+- Zebra's continuous integration checks that tier 1 targets will always build
+  and pass tests.
+
+Adding a new tier 3 target imposes minimal requirements; but we focus primarily
+on avoiding disruption to other ongoing Zebra development.
+
+Tier 2 and tier 1 targets place work on Zebra project developers as a whole, to
+avoid breaking the target. These tiers require commensurate and ongoing efforts
+from the maintainers of the target, to demonstrate value and to minimize any
+disruptions to ongoing Zebra development.
+
+This policy defines the requirements for accepting a proposed target at a given
+level of support.
+
+Each tier is based on all the requirements from the previous tier, unless
+overridden by a stronger requirement.
+
+While these criteria attempt to document the policy, that policy still involves
+human judgment. Targets must fulfill the spirit of the requirements as well, as
+determined by the judgment of the approving teams. Neither this policy nor any
```

Comment:
There is only one Zebra team, so it's a bit confusing to talk about multiple teams.

Please make these changes wherever a team or teams are mentioned, throughout the document.

```suggestion
determined by the judgment of the Zebra team. Neither this policy nor any
```